packages/suite/src/constants/wallet/sendForm.ts

```diff
@@ -47,3 +47,5 @@ export const DEFAULT_VALUES = {
     rippleDestinationTag: '',
     outputs: [],
 } as const;
+
+export const CARDANO_DEFAULT_TTL = 7200;
```

Comment:
I understand I am starting to be annoying with this, ...but.. could we have links to cardano documentation that defines these values? And this comment applies in general. Whenever there is a constant I would be happy if it:
- could be defined in some constants file
- it would have a comment with url to documentation if possible.

Reply:
This is arbitrary value we've chosen, yoroi, adalite, nami wallet all use 1h or 2h (7200s). I'll add a comment there.